core/utils.go

```diff
@@ -114,37 +114,48 @@ func ValidBumpLevel(level string) bool {
 		level == AutoLevel
 }
 
-func HighestAutoLevel(config *Config, allChanges []Change) (string, error) {
+func HighestAutoLevel(config *Config, allChanges []Change) (highestLevel string, err error) {
```

Comment:
```suggestion
func HighestAutoLevel(config *Config, allChanges []Change) (string, error) {
```
No need for return variables here, can just create variables down below.